lib/twilio.rb

```diff
@@ -0,0 +1,20 @@
+require 'rubygems' # not necessary with ruby 1.9 but included for completeness
+require 'twilio-ruby'
+require_relative 'customer'
+
+def timer
+  date = Time.now + (3600)
+  delivery_time = date.to_s.split(' ')[1]
+end
+
+def send_text(name, phone)
+  account_sid = 'AC2f4d6a3fb2fcdd3e058b6930b7f47697'
+  auth_token = '18a596c4b02371cb565004ab54694bcf'
+  @client = Twilio::REST::Client.new account_sid, auth_token
+
+  message = @client.account.messages.create(
+  :body => "#{name}Your food is on its way! It will be there by #{timer}" ,
```

Comment:
Indent the first parameter one step more than the previous line.<br>Use the new Ruby 1.9 hash syntax.<br>Space found before comma.